### DISCUSSION TITLE
Eliminating redundancy in the default feed

### DISCUSSION THREAD

**Discussion by venteto** [bluesky-social/atproto#1998] [Bluesky Client App]
Firstly, it would be more useful to never see my own posts in my own default feed. If I ever want to see my own posts, all I have to do is visit my _profile_.

Secondly, reposts by others could benefit from multiple improvements.

If a database could record whether any given post has been interacted with by me, or even just seen by me, then there would never be any reason to show me a repost of that post, whether by the person who originally posted it, or any other account, regardless of whether I follow any of those accounts. It's good that  Bluesky is testing a feature to allow hiding of posts through manual intervention, but providing a _settings option_ whereby I could choose to not see reposts of posts I've already seen would be a useful way of auto-hiding redundant posts. The settings option could either be a simple on/off toggle, or something with a selected threshold, e.g., I've scrolled by the post, I've expanded the post, I've liked the post, (hopefully in the future) I've bookmarked the post, I've replied to the post, I've manually hidden the (original) post, or I've reposted or quote posted the post myself.

**Comment by snarfed** [comment]
Sounds like job(s) for a custom feed!

**Comment by venteto** [comment]
I think it's a job for _sane defaults_ (see #2002)
